pet-finder/src/Components/Forms/LoginForm/index.tsx

```diff
@@ -16,28 +24,48 @@ export const LoginForm: FC<LoginFormProps> = () => {
     register,
     handleSubmit,
     formState: { errors, isValid },
-  } = useForm({ mode: 'all', resolver: yupResolver(loginSchema()) })
-  const onSubmit: SubmitHandler<LoginFormType> = ({ email, password }) => {
-    console.log(email, password)
-  }
+    setError,
+  } = useForm({
+    mode: 'all',
+    resolver: yupResolver(loginSchema(ERROR_MESSAGES)),
+  })
+
+  const onSubmit: SubmitHandler<LoginFormType> = useCallback(
+    async ({ email, password }) => {
+      console.log(email, password)
+      try {
+        const response = await logIn({ email, password })
+        if (!response.ok) {
+          const serverMessage =
+            exceptionResponse[response.status as keyof ExceptionMessage]
+          setError('root.serverError', { message: serverMessage })
+        } else {
+          const responseBody: IResponseBodyLogIn = await response.json()
+          console.log(responseBody)
```

Comment:
same, console log

Reply:
Lets leave this logs, work is not finished here

Reply:
Lets leave this log, work in not finished here